src/editor.js

```diff
@@ -383,6 +383,27 @@ const PreviewPanel = ({
   </div>
 `;
 
+const dispatchUpdatePreview = redispatchAs(g('update-preview'));
+
+const RefreshIcon = () => html`
+  <div @click=${dispatchUpdatePreview}>
```

Comment:
give this a class `n('refresh-button')` so you can use that selector instead of `.dispatchUpdatePreview`.